packages/search-engine/__tests__/components/Filters/ModalFilters.test.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import ModalFilters from '../../../src/components/Filters/ModalFilters'
+
+describe('Given that ModalFilter component is used', () => {
+  test('When the variable visible is false, then the modal should be invisible', () => {
+    const title = 'modalTitle'
+    const children = 'Texto de prueba'
```

Comment:
@simontorresdev what do you think if instead of hardcoding values we use libraries such as Faker to create these values?